apps/condo/domains/settings/schema/MobileFeatureConfig.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyGreaterThanPreviousMeterReadingIsEnabled:Checkbox; meta:Json; ticketSubmittingIsDisabled:Checkbox'`
+ */
+
+const { Text, Relationship, Checkbox } = require('@keystonejs/fields')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/settings/access/MobileFeatureConfig')
+
+
+
+const ERRORS = {
+    TICKET_SUBMITTING_PHONES_NOT_CONFIGURED: {
+        code: BAD_USER_INPUT,
+        type: 'TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+        message: 'commonPhone field not specified',
+        messageForUser: 'api.organization.MobileFeatureConfig.TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+    },
+    COMMON_PHONE_INVALID: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'commonPhone'],
+        type: 'COMMON_PHONE_NOT_CONFIGURED',
+        message: 'commonPhone is invalid',
+        messageForUser: 'api.organization.MobileFeatureConfig.COMMON_PHONE_INVALID',
+    },
+    ORGANIZATION_IS_REQUIRED: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'organization'],
+        type: 'ORGANIZATION_IS_REQUIRED',
+        message: 'The organization is required to create',
+        messageForUser: 'api.organization.MobileFeatureConfig.ORGANIZATION_IS_REQUIRED',
+    },
+}
+
+const MobileFeatureConfig = new GQLListSchema('MobileFeatureConfig', {
+    schemaDoc: 'Manages availability of some features in mobile application,' +
+        ' stores data required in disabled state of a feature.',
+    fields: {
```

Comment:
It's not understandable how auxiliary fields belong to feature (enabled/disabled) areas.

Reply:
This data is necessary for the mobile application to determine what to display. In the case of the feature of disabling tickets - displaying the desired phone.
If you are describing the full changes that will occur in the mobile app when you disable/enable a feature, it will mean that this place must be maintained when changes are made in the mobile app, otherwise this data source will mislead the backend developer.

Reply:
>> What about using the same prefix for auxiliary field naming?

that's how it happens. All fields required for feature management are marked with a prefix. That is why the `ticketSubmittingIsDisabled` field is called so incorrectly (from the point of view of the English language). Pahaz asked for some fields, for example, the org phone, which will be reused with a high probability, not to attach to a specific feature, but to leave commonPhone